tools/python/maps_generator/var/etc/map_generator.ini.default

```diff
@@ -1,70 +1,74 @@
-[Main]
-# If the flag DEBUG is set a special small planet file will be downloaded.
-DEBUG: 1
-# The path where the planet will be downloaded and the maps are generated.
-MAIN_OUT_PATH: ~/maps_build
-# The path where caches for nodes, ways, relations are stored.
-# CACHE_PATH:
+[Developer]
+# Path to the `organicmaps` source code repository:
+OMIM_PATH: ~/OM/organicmaps
+# A path with the generator_tool binary:
+BUILD_PATH: ${Developer:OMIM_PATH}/../omim-build-release
 
 
-[Developer]
-# The path where the generator_tool will be searched.
-BUILD_PATH: ~/omim-build-release
-# The path to the project directory omim.
-OMIM_PATH: ~/omim
+[Main]
+# A special small planet file will be downloaded if DEBUG is set to 1.
+DEBUG: 0
```

Comment:
Remove this option, like it was removed in code.

Reply:
Its in the code still (it was duplicated there and I removed one duplicate that was causing an unexpected behavior also).